<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="HomePage">
     <widget class="QLabel" name="BolusIcon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="BolusBar">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="BolusNum">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OptionsButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>OPTIONS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BolusButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>271</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>BOLUS</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>431</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="OffPage">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>561</width>
        <height>481</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;OFF&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QFrame" name="PINFrame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>PIN:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="PIN">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ErrorPIN">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>60</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="OptionsPage">
     <widget class="QPushButton" name="ConfigurationButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ConfigPage">
     <widget class="QGraphicsView" name="DTView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="PINView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>541</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DTLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Date / Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="PINLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>241</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PIN (Personal Identification Number)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Current PIN:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CurrentPIN">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>310</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="DeletePINButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete PIN</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="NewPIN">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SetPINButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>410</y>
        <width>83</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Set PIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>370</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>New PIN:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ErrorNewPIN">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="OnButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OffButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChargeButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Charge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UnplugButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Unplug</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="BatteryBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="TandemLogo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>111</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string>Tandem</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
